util/formatChartTimes.ts

```diff
@@ -0,0 +1,53 @@
+import { NumberValue } from 'd3'
+
+const createTimeFormatter = (timestamp: NumberValue, locale: string, options: Intl.DateTimeFormatOptions) =>
```

Comment:
I think all these utils should be generic and accept dates. there should not be d3 dependent. 
Then you should have a util that convert from NumberValue to Date